src/transformers/tokenization_roberta.py

```diff
@@ -28,15 +28,15 @@
 }
 
 PRETRAINED_VOCAB_FILES_MAP = {
-    "vocab_file": {
+    "vocab_file": {  # These have identical contents
```

Comment:
It's ok for us to have some level of duplicated weights/vocab. It allows a clearer understanding for the users (similar to our permissive policy on code duplication) and cleaner statistics on the model pages.

Reply:
Yeah, not opposed. I just want future reader to understand that there is no difference.